roles/openshift_logging_elasticsearch/tasks/main.yaml

```diff
@@ -106,6 +106,15 @@
     resource_name: system:auth-delegator
     user: "system:serviceaccount:{{ openshift_logging_elasticsearch_namespace}}:aggregated-logging-elasticsearch"
 
+- oc_adm_policy_user:
+    state: present
+    namespace: "{{ openshift_logging_elasticsearch_namespace }}"
+    resource_kind: scc
+    resource_name: privileged
```

Comment:
Does ES require `privileged` SCC? Shouldn't `hostmount-anyuid` suffice?

Reply:
I also think that "privileged" is over relaxed constraint. But I followed the documentation: 

https://docs.okd.io/3.11/install_config/aggregate_logging.html#aggregated-logging-hostPath-Elasticsearch

6. Give the privilege to mount and edit a local volume to the relevant service account:

```
$ oc adm policy add-scc-to-user privileged  \
         system:serviceaccount:logging:aggregated-logging-elasticsearch
```

Reply:
es does not need privileged  - use hostmount-anyuid - this is how we do it in viaq: https://github.com/ViaQ/Main/blob/master/README-install.md#persistent-storage and https://github.com/ViaQ/Main/blob/master/README-install.md#enabling-elasticsearch-to-mount-the-directory

Reply:
sounds like we need to update the docs